### ISSUE TITLE
Information for genesis block(Timestamp incorrect)

### ISSUE THREAD

**Issue by AyushyaChitransh** [ethereum/go-ethereum#17042]
#### System information

Geth version: 

> Geth
> Version: 1.8.3-unstable
> Git Commit: a06387674983d93453b800380f6f55c6406f51e3
> Architecture: amd64
> Protocol Versions: [63 62]
> Network Id: 1
> Go Version: go1.9.2
> Operating System: linux
> GOPATH=
> GOROOT=/usr/lib/golang

#### Expected behaviour

Timestamp for block 0 is 1438269973, which translates to Jul-30-2015 03:26:13 PM +UTC

#### Actual behaviour

Timestamp for block 0 is 0, which translates to Thursday, January 1, 1970 12:00:00 AM

#### Steps to reproduce the behaviour

Sync mainnet and once sync is complete, query the details of block 0.

**Comment by gsalzer** [comment]
The same holds when querying a node running Parity: timestamp for block 0 is 0. On the other hand, etherscan.io shows `Jul-30-2015 03:26:13 PM +UTC` as timestamp for the genesis block. It would be nice to have a consistent view, or at least an explanation like "0 is correct, etherscan is just extrapolating".

**Comment by mxjxn** [comment]
Has this been confirmed by anyone here? Seems like a pretty serious issue. When I create a private chain, the first block is by default 1/1/1970. I've confirmed this on multiple machines. Affects downstream tools like truffle / ganache.

**Comment by holiman** [comment]
So, etherscan does show the genesis: https://etherscan.io/block/0 -- but they must have put some hack into place to make it show up as ` (Jul-30-2015 03:26:13 PM +UTC) ` -- because AFIK the actual timestamp on the genesis is all zeroes, which is 1970. Unless I'm mistaken, there's no error in geth here, possibly etherscan.

**Comment by adamschmideg** [comment]
Here is the [original genesis.json](https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/resources/genesis/frontier.json#L26688) that says `"timestamp": "0x00"`.
